### ISSUE TITLE
[MAS4.1.2] Name property does not defined  for “Date" and "Time" editable text boxes from all card inputs data.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1347]
**Actual Result:**
When focus lands on “Date" and "Time" editable text boxes then narrator is not reading name, only reading values which showing on the text box.
Name property does not defined for “Date" and "Time" editable text boxes.

**Note** - Same issue repro at - 
1. URL -> Help -> Receipt card -> Name property does not define properly for controls which define as a button. Attachment "MAS4.1.2_Name for controls define as button".

**Expected Result:**
Name property should be defined properly for  “Date" and "Time" editable text boxes.

**Repro Steps:**

1. Open URL [https://microsoft.github.io/BotFramework-WebChat/full-bundle/](https://microsoft.github.io/BotFramework-WebChat/full-bundle/) in Edge browser.
2. Notice 'Upload File' and 'Send' button icons present in bottom of screen.
3. Type 'Help' in the input control at the bottom of the screen and hit enter (or click on send button).
4. Content will upload, Navigate to any "Show an Adaptive cards with all types of inputs " button by using Tab key and press Enter on it.
5.  New data will upload.
6. Open Narrator (Ctrl + Win. key + Enter)
7. Navigate to “Date" and "Time" editable text boxes by using Tab key.
8. Observe that Name property does not defined for “Date" and "Time" editable text boxes.

**User Impact:**
If Name,State, Value and Role property is not defined properly for controls then the users will get confuse about the purpose of that control.

**MAS Reference:**
[MAS4.1.2 - Name,Role,Value](https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730})

**Test Environment:**
**OS:** Windows 10
**OS Build:** 17134.345
**OS Version:** 1803
**Browser:** Edge
**Tool:** Narrator

**Attachment:**
1. [MAS4.1.2_Name for date and time controls.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2579227/MAS4.1.2_Name.for.date.and.time.controls.zip)
2. ![mas4 1 2_name for controls define as button](https://user-images.githubusercontent.com/41668935/49133374-db093580-f305-11e8-8ddc-3a2b7af95f6b.png)

**Comment by compulim** [comment]
This looks like a bug on Adaptive Cards. Today, they don't have a JSON property to put the name on [their schema](https://docs.microsoft.com/en-us/adaptive-cards/authoring-cards/card-schema#inputdate).

Please file this bug to Adaptive Cards.

**Comment by compulim** [comment]
Reopening so we can confirm from the accessibility team.

**Comment by corinagum** [comment]
Confirmed as external with accessibility team

**Comment by Amit8527510735** [comment]
#HCL;#Accessibility;#HCL_BotFramework_WebChat;A11yMAS;WCAG4.1.2
